terraform/concourse/concourse.tf

```diff
@@ -159,7 +159,9 @@ resource "terraform_data" "provision_concourse" {
     command = <<-CMD
       gcloud compute ssh ${var.hostname} --zone ${var.zone} --tunnel-through-iap --command "\
       sudo /concourse/scripts/install.sh ${var.docker_ver} ${var.data_disk} ;\
-      sudo /concourse/scripts/start.sh ${var.hostname}.${var.dns_zone_fqdn} ${var.force_new_cert}" --verbosity=error
+      sudo /concourse/scripts/start.sh ${var.hostname}.${var.dns_zone_fqdn} ${var.force_new_cert} ;\
+      // cron tab runs every weekend e.g. on Sunday at midnight ;\
+      sudo ln -nfs /concourse/scripts/prune_workers.sh /etc/cron.weekly/prune_workers" --verbosity=error
```

Comment:
```suggestion
      sudo ln -nfs /concourse/scripts/prune_workers.sh /etc/cron.weekly/prune_workers"
```
the `verbosity=error` was a leftover from the previous code.

The comment above is also wrong now.
From the `/etc/crontab` file on a Jumpbox (Ubuntu Jammy):
```
i551749@cf-zz86-test-jumpbox:~$ cat /etc/crontab
# /etc/crontab: system-wide crontab
# Unlike any other crontab you don't have to run the `crontab'
# command to install the new version when you edit this file
# and files in /etc/cron.d. These files also have username fields,
# that none of the other crontabs do.

SHELL=/bin/sh
# You can also override PATH, but by default, newer versions inherit it from the environment
#PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin

# Example of job definition:
# .---------------- minute (0 - 59)
# |  .------------- hour (0 - 23)
# |  |  .---------- day of month (1 - 31)
# |  |  |  .------- month (1 - 12) OR jan,feb,mar,apr ...
# |  |  |  |  .---- day of week (0 - 6) (Sunday=0 or 7) OR sun,mon,tue,wed,thu,fri,sat
# |  |  |  |  |
# *  *  *  *  * user-name command to be executed
17 *	* * *	root    cd / && run-parts --report /etc/cron.hourly
25 6	* * *	root	test -x /usr/sbin/anacron || ( cd / && run-parts --report /etc/cron.daily )
47 6	* * 7	root	test -x /usr/sbin/anacron || ( cd / && run-parts --report /etc/cron.weekly )
52 6	1 * *	root	test -x /usr/sbin/anacron || ( cd / && run-parts --report /etc/cron.monthly )
```

so it would be on Sunday at 6:47, not midnight. But I would just leave that out. It runs weekly and that's clear from the `/etc/cron.weekly` folder.

Reply:
The `--verbosity=error` is include for whole command which include running other scripts e.g. `install.sh` and `start.sh`. I would leave it as is.